queries/vimdoc/highlights.scm

```diff
@@ -33,13 +33,13 @@
 
 (codeblock
   [
-    ">"
-    (language)
-  ] @markup.raw.delimiter
+    ">" @markup.raw
+    (language) @markup.environment.name
+  ] @markup.raw
   (#set! conceal ""))
```

Comment:
I know this works, but probably better if we split this into two. Reading this seems awkward because `(language)` has 2 captures for it

Reply:
Whoops, this was me changing my mind mid-change. The grouping needs to go of course.